### ISSUE TITLE
Request for Key Term description for 'repair techniques'

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/wcag#2563]
Hi I have been asked to clarify the term Repair Techniques as mentioned in the document '[Understanding Success Criterion 4.1.1: Parsing](https://www.w3.org/WAI/WCAG21/Understanding/parsing)'. 

"Since **repair techniques** vary among user agents, authors cannot assume that content will be accurately parsed into a data structure or that it will be rendered correctly by specialized user agents, including assistive technologies..."

Could this be added to a 'Key Terms' for the 4.1.1 Parsing understanding document.

**Comment by bruce-usab** [comment]
Thank you for raising this question!

Clarifying edits to Understanding are most welcome.  We have "key terms" in other Understanding pages, so adding that section for this one seems like a reasonable approach.  I am not sure that goes quite far enough.  I think it might be worth some discussion so as to tease out better what is meant by the concept.

For starters, since we use Techniques for human author best practices, it strikes me as confusing to be using that word here for automated machine browser error recovery.

@LaurenceRLewis do you have a proposal for something like a glossary term?

**Comment by patrickhlauke** [comment]
glossary terms are part of the normative document though, so not limited to understanding.

In general though, I think rather than making up a term definition, it may be easier to just make that sentence itself a bit more immediately readable/understandable. e.g.

"Since the way user agents will attempt to parse and error-correct malformed or invalid markup may vary among user agents, authors..."

**Comment by LaurenceRLewis** [comment]
@patrickhlauke I think updating the content to make the sentence more immediately readable is a good idea. Readers then do not have to relocate on the page to understand the term. 

"Since the way user agents will attempt to parse and error-correct malformed or invalid markup may vary among user agents, authors..."

**Comment by patrickhlauke** [comment]
Made an initial stab here https://github.com/w3c/wcag/pull/2566

**Comment by patrickhlauke** [comment]
Closing, as #2566 was closed
